### PR TITLE
refactor(button): apply latest specs - INNO-1500

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Apply [latest specs](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Button+Design+Specifications) for buttons

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 076ae8c54c245b5cbf5af233085c1f42f23dbab7

https://deploy-preview-1135--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit e40b30afb6d66ca6f24060bb117fc374c0c41b98

https://deploy-preview-1135--europa-component-library.netlify.com

**Review by yhuard — CHANGES_REQUESTED** [review]
The new specs look properly applied.

There's still "Ghost button" in the docs ("Button name" column), could you update it? Thanks

**Review by yhuard — APPROVED** [review]

File: src/systems/eu/implementations/react/components/button/stories/Index.jsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ storiesOf('Components|Button', module)
 
     return (
       <Button
-        variant="ghost"
+        variant="text"
```
Comment (yhuard) [inline src/systems/eu/implementations/react/components/button/stories/Index.jsx]: the variant is still `ghost`

File: src/systems/ec/specs/components/button/demo/data--ghost.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Simple content for demo
 module.exports = {
-  label: 'Ghost button',
+  label: 'Text button',
```
Comment (yhuard) [inline src/systems/ec/specs/components/button/demo/data--ghost.js:3]: you forgot to rename it in the EU system as well

**Review by yhuard — CHANGES_REQUESTED** [review]

**Review by yhuard — APPROVED** [review]

**Pull request merged**